api/src/main/java/org/openmrs/module/fhir2/FhirConstants.java

```diff
@@ -112,7 +112,10 @@ private FhirConstants() {
 	public static final String OPENMRS_FHIR_EXT_OBSERVATION_REFERENCE_RANGE = OPENMRS_FHIR_EXT_PREFIX
 	        + "/obs/reference-range";
 	
-	public static final String OPENMRS_FHIR_EXT_RECORDED = OPENMRS_FHIR_EXT_PREFIX + "/dispense/recorded";
+	public static final String OPENMRS_FHIR_EXT_RECORDED = OPENMRS_FHIR_EXT_PREFIX + "/medicationdispense/recorded";
```

Comment:
This is tangential, it seemed like "medicationdispense" was the more proper name than "dispense"?